### ISSUE TITLE
TTS Respnse 

### ISSUE THREAD

**Issue by rgood68** [k4l1sh/alexa-gpt#19]
Hello, I have created a skill in Portuguese (Brazil), and I received the following error message:

"[The skill's TTS response should be presented in a language supported by Alexa and match the locale of the skill.](https://developer.amazon.com/alexa/console/ask/certify/amzn1.ask.skill.dc129adb-e20e-4f0e-b457-7cccd072f2b0/development/pt_BR/validation#)
The skill's TTS response should be presented in a language supported by Alexa and matches the locale of the skill. If the skill supports multiple locales, the TTS responses for each locale has to be presented in the relevant language. The following TTS responses were found in a language that does not match the skill's locale.
Response : Leaving Chat G. P. T. mode, TTS Language Found: English
Response : Vaginal Davis is an American artist, performer, and musician known for her work in the fields of performance art, music, and film. She is a pioneering figure in the queer and punk scenes, challenging societal norms and pushing boundaries with her provocative and unique artistic expressions., TTS Language Found: English
Response : Chat G. P. T. mode activated, TTS Language Found: English
Response : Brenda Marshall was an American actress known for her roles in classic films such as "The Sea Hawk" and "The Constant Nymph. " She was active in Hollywood during the 1940s and 1950s, often appearing in supporting roles alongside top stars of the era., TTS Language Found: English
Response : Sorry, I had trouble doing what you asked. Please try again., TTS Language Found: English"

I have no idea how to fix this. Could you help, please?

**Comment by rgood68** [comment]
Bhy the way, I could not find the lambda function at my AWS Lambda dashboard, even changing the region

**Comment by rgood68** [comment]
I have just seem a tutorial that says that it is necessary to create a lambda function manually and upload a zip file. What is the content of such zip file?

**Comment by k4l1sh** [comment]
This lambda function is the lambda_function.py inside this repository, but you don't need to do it to create alexa skills, these things are already built-in.

Did you created the skill the console ask webpage? https://developer.amazon.com/alexa/console/ask
The Portuguese (BR) is available.
![portuguese](https://i.imgur.com/Kkkddf6.png)

**Comment by rgood68** [comment]
Yes, I am using ask console. I'm confused, e.g. I couldn't find in this lambda function from the tutorial where to change the version of the GPT chat as mentioned in another issue. I searched here and understood that I would have to change this in the lambda function in AWS, is it not necessary? Where do I change the GPT chat version?
